core/testlib/Test/Service/EventStore/GlobalStreamOrdering/Spec.hs

```diff
@@ -75,9 +75,7 @@ spec newStore = do
           context.store.readAllEventsForwardFrom (Event.StreamPosition 0) (EventStore.Limit (expectedTotalEvents))
             |> Task.mapError toText
         Task.unless ((Array.length allGlobalEvents) <= 1) do
-          let eventPairs =
-                allGlobalEvents
-                  |> Array.zip (Array.drop 1 allGlobalEvents)
+          let eventPairs = Array.zip allGlobalEvents (Array.drop 1 allGlobalEvents)
```

Comment:
Can you modify `Array.zip` so this modification is not needed? The idea is that we dont touch the tests.

Remember that in NeoHaskell, the Array argument (this/self in other languages) must go **last**, contrary to the regular functions in Haskell.

Once this is solved, I'll merge this :)

Reply:
Done! :)